tezos.go

```diff
@@ -0,0 +1,37 @@
+package coincodec
+
+import (
+	"github.com/wealdtech/go-slip44"
+	"errors"
+)
+
+const (
+	AddressLength = 23
+)
+
+func init() {
+	toBytesMap[slip44.TEZOS] = TezosDecodeToBytes
+	toStringMap[slip44.TEZOS] = TezosEncodeToString
+}
+
+// TezosDecodeToBytes converts the input string to a byte array
+func TezosDecodeToBytes(input string) ([]byte, error) {
+	decoded, err := Base58Decode(input, Base58DefaultAlphabet)
```

Comment:
for default alphabet, let's use `Decode` in `btcutil/base58`

Reply:
For Tezos we need default alphabet, and checksum.  The btcsuite/btcutil uses default alphabet, and has a checksum version, but that also uses one byte version.  So I don't think it's a good idea to use btcsuite/btcutil for Tezos.